### DISCUSSION TITLE
Integrating to easy-random fork

### DISCUSSION THREAD

**Discussion by dvgaba** [murdos/easy-random-protobuf#127] [General]
Hi,

Hope you are doing great. I really liked your project and  I am thinking to integrate your project in my easy-random fork. given the increasing usage of protobuf. Let me know your thoughts on the same.

For reference 

https://github.com/dvgaba/easy-random [JDK16]
https://github.com/dvgaba/easy-random-jdk11

Regards
DG

**Comment by murdos** [comment]
Hi,

Why do you want to integrate easy-random-protobuf into your easy-random fork?
Easy-random is built with extension and modularity in mind, I don't think there's a need for grouping together all easy-random generators.

**Reply by dvgaba** [reply]
It is easier to manage and extend under a single umbrella for example I made some changes in existing predicates design to have better extensibility.

**Comment by murdos** [comment]
Closing since @dvgaba has forked the code from this repository to its own.
Note that code has diverged and doesn't offer the same features or fixes.
